canonical-web-and-design/maas-ui
# List pools

## Done
Add list of pools.

## QA
Visit /pools. A table should display pools correctly, with action buttons in the appropriate state (disabled without permissions, only edit for the default pool).

![image](https://user-images.githubusercontent.com/130286/73719218-938cfd80-4783-11ea-878b-61568b2db5fa.png)
